docs/env-vars.md

```diff
@@ -30,6 +30,7 @@
 | `OPENAI_API_KEY`                         | OpenAI access key                                                                   |                                         |
 | `OPENAI_API_BASE`                        | To set the base URL path for API requests                                           | `https://api.openapi.com/v1`            |
 | `OPENAI_MODEL`                           | OpenAI model                                                                        | `gpt-4o`                                |
+| `TRUSTD_ADDITIONAL_CLIENTS`              | Additional allowed development OAuth clients                                        |                                         |
```

Comment:
As this is only available in devmode. Maybe rename it to `TRUSTD_DEVMODE_ADDITIONAL_CLIENTS`. Do make it very explicit.